docs/workforce-management-framework/topics/con-passportjs-module.adoc

```diff
@@ -0,0 +1,17 @@
+[id='con-passportauth-module-{chapter}']
+= Introducing the {WFM-RC-NameShort} Passport.js Module
+
+*Passport.js* is a lightweight authentication middleware for Node.js that is extremely flexible and modular.
+*Passport.js* offers a comprehensive set of link:http://passportjs.org/docs/configure[strategies] that support authentication using a username and password, Facebook, Twitter, and more(?).
+
+*Passport.js*:
+
+* Can be bundled into an existing Node.js server
+* Is available by default when running our reference server application(?)
```

Comment:
What is the "our reference server application(?)" - it's not mentioned anywhere else

Reply:
its a reference to the demo app. Demo app will not exist going forward as a product so we may have to figure out what to call it.